scenes/PlanetSurface/PlanetSurface.gd

```diff
@@ -5,7 +5,7 @@ var planet_data = null
 
 func _ready() -> void:
 	Surface_start.grab_focus()
-	$CanvasLayer/PlanetPanel/RichTextLabel.text = planet_data.name
+	$CanvasLayer/PlanetPanel/RichTextLabel.text = planet_data.shipyard_description
```

Comment:
Should we do something like `if "shipyard_description" in planet_data:` for this and the other fields just in case they are not set in the systems.json?

Other than that everything looks good!